subliminal/converters/opensubtitlescom.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
```

Comment:
AFAIK this is the default with modern Python versions

Reply:
Yes, we can remove these lines from all the files as part of #1071